po/ro.po

```diff
@@ -0,0 +1,703 @@
+
+#: src/install.rs:1965
+msgid "{}-{} is up to date -- skipping install"
+msgstr "{}-{} este actualizat la ultima versiune -- se sare peste instalare"
+
+#: src/keys.rs:47
+msgid "keys need to be imported:)"
+msgstr "cheile trebuie importate:)"
+
+#: src/keys.rs:52
+msgid "     {key} wanted by: {base}"
+msgstr "     {key} dorită de: {base}"
+
+#: src/news.rs:57
+msgid "No Date "
+msgstr "Nicio Dată"
+
+#: src/news.rs:60
+msgid "No Title"
+msgstr "Niciun Titlu"
+
+#: src/news.rs:69
+msgid "no new news"
+msgstr "nicio știre nouă"
+
+#: src/query.rs:105
+msgid " [ignored]"
+msgstr " [ignorat]"
+
+#: src/repo.rs:164
+msgid "failed to get current exe"
+msgstr "nu s-a putut obține current exe"
+
+#: src/repo.rs:195
+msgid "syncing local databases..."
+msgstr "se sincronizează baza de date locală..."
+
+#: src/repo.rs:201
+msgid "  nothing to do"
+msgstr " nimic de făcut"
+
+#: src/search.rs:32
+msgid "aur search failed"
+msgstr "căutarea aur a eșuat"
+
+#: src/search.rs:158
+msgid "[Out-of-date: {}]"
+msgstr "[Expirat: {}]"
+
+#: src/search.rs:164 src/search.rs:218
+msgid "[Installed: {}]"
+msgstr "[Instalat: {}]"
+
+#: src/search.rs:166 src/search.rs:220
+msgid "[Installed]"
+msgstr "[Instalat]"
+
+#: src/search.rs:173
+msgid "[Orphaned]"
+msgstr "[Orphaned]"
+
+#: src/search.rs:263
+msgid "no packages match search"
+msgstr "niciun pachet nu se potrivește cu căutarea"
+
+#: src/search.rs:321
+msgid "Packages to install (eg: 1 2 3, 1-3):"
+msgstr "Pachetele care vor fi instalate (eg: 1 2 3, 1-3):"
+
+#: src/stats.rs:83
+msgid "Total installed packages: {}"
+msgstr "Toate pachetele instalate: {}"
+
+#: src/stats.rs:87
+msgid "Aur packages: {}"
+msgstr "Pachete Aur: {}"
+
+#: src/stats.rs:91
+msgid "Repo packages: {}"
+msgstr "Pachete Repo: {}"
+
+#: src/stats.rs:95
+msgid "Explicitly installed packages: {}"
+msgstr "Pachete instalate explicit: {}"
+
+#: src/stats.rs:99
+msgid "Total Size occupied by packages: {}"
+msgstr "Spațiul Total ocupat de pachete: {}"
+
+#: src/stats.rs:106
+msgid "Ten biggest packages:"
+msgstr "Cele mai mari zece pachete:"
+
+#: src/sync.rs:89 src/lib.rs:395
+msgid " [installed]"
+msgstr " [instalate]"
+
+#: src/upgrade.rs:122
+msgid "Looking for AUR upgrades"
+msgstr "Se caută upgrade-uri AUR"
+
+#: src/upgrade.rs:148
+msgid "Looking for devel upgrades"
+msgstr "Se caută upgrade-uri devel"
+
+#: src/upgrade.rs:179
+msgid "warning:"
+msgstr "avertizare:"
+
+#: src/upgrade.rs:181
+msgid "{pkg}: ignoring package upgrade ({old} => {new})"
+msgstr "{pkg}: se ignora upgrade-ul pachetului ({old} => {new})"
+
+#: src/upgrade.rs:334
+msgid "Packages to exclude (eg: 1 2 3, 1-3):"
+msgstr "Pachetele care vor fi excluse (eg: 1 2 3, 1-3):"
+
+#: src/util.rs:80
+msgid "[Y/n]:"
+msgstr "[Y/n]"
+
+#: src/util.rs:82
+msgid "[y/N]:"
+msgstr "[y/N]"
+
+#: src/util.rs:101
+msgid "y"
+msgstr "y"
+
+#: src/util.rs:101
+msgid "yes"
+msgstr "yes"
+
+#: src/util.rs:103
+msgid "n"
+msgstr "n"
+
+#: src/util.rs:103
+msgid "no"
+msgstr "no"
```

Comment:
Shouldn't this be nu?

Reply:
Yes it should, thank you, I actually missed these when I made a second pass to check my translations. It's fixed now, for both no and yes.